hackathon_site/dashboard/frontend/src/pages/Inventory/Inventory.js

```diff
@@ -5,137 +5,177 @@ import Drawer from "@material-ui/core/Drawer";
 import Header from "components/general/Header/Header";
 import Typography from "@material-ui/core/Typography";
 import TextField from "@material-ui/core/TextField";
-import useMediaQuery from "@material-ui/core/useMediaQuery";
 import Divider from "@material-ui/core/Divider";
 import IconButton from "@material-ui/core/IconButton";
 import Hidden from "@material-ui/core/Hidden";
 import Button from "@material-ui/core/Button";
 import RefreshIcon from "@material-ui/icons/Refresh";
 import SearchIcon from "@material-ui/icons/Search";
+import CloseIcon from "@material-ui/icons/Close";
 import FilterListIcon from "@material-ui/icons/FilterList";
 import Grid from "@material-ui/core/Grid";
 import EnhancedInventoryFilter from "components/inventory/InventoryFilter/InventoryFilter";
 import Item from "components/inventory/Item/Item";
-import { inventoryItems } from "testing/mockData";
+import ProductOverview from "components/inventory/ProductOverview/ProductOverview";
+import { productInformation, inventoryItems } from "testing/mockData";
 
 const Inventory = () => {
     const [mobileOpen, setMobileOpen] = React.useState(false);
-
     const toggleFilter = () => {
         setMobileOpen(!mobileOpen);
     };
 
-    const mobileWidth = useMediaQuery(useTheme().breakpoints.up("md"));
-
+    // Remove this later once filter data is able to be submitted
     const sleep = (ms) => new Promise((r) => setTimeout(r, ms));
     const onSubmitTemp = async ({ orderBy, inStock, inventoryCategories }) => {
         await sleep(300);
         alert(JSON.stringify({ orderBy, inStock, inventoryCategories }, null, 2));
+        setMobileOpen(false);
+    };
+
+    // Remove this later once items can be added to cart
+    const addToCart = () => {
+        alert("Add to cart");
+        setItemOverview(null);
+    };
+
+    const [itemOverview, setItemOverview] = React.useState(null);
+    const toggleMenu = () => {
+        setItemOverview(null);
     };
 
+    // itemOverview will be sent to api
```

Comment:
Shouldn't need to send anything to the API since we should already have all the information in Redux, but it will dispatch an action to show the overview drawer for the appropriate item id (which will then pull what's needed from Redux)